### PR TITLE
Step 20 Updates

### PR DESCRIPTION
**Pull request by dotansimha**

### CODE REVIEW

File: guides/git.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## Working with git
```
Comment (Urigo) [inline guides/git.md:1]: @dotansimha what's that?

File: COC.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Code of Conduct
```
Comment (Urigo) [inline COC.md:1]: @dotansimha those should come only in master, not here

File: README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# meteor-angular-socially - [Tutorial's site](http://angularjs.meteor.com/tutorial)
```
Comment (Urigo) [inline README.md:1]: @dotansimha those should come only in master, not here

**Comment by Urigo** [comment]
@dotansimha is this still relevant?

**Comment by dotansimha** [comment]
@Urigo  No

**Pull request closed without merging**